doc/releases/changelog-dev.md

```diff
@@ -37,6 +37,29 @@
 * New basis sets, `6-311g` and `CC-PVDZ`, are added to the qchem basis set repo.
   [#3279](https://github.com/PennyLaneAI/pennylane/pull/3279)
 
+* Added a new gate operation `FermionicSWAP`, which implements the exchange of spin orbitals
+  representing fermionic-modes while maintaining proper anti-symmetrization.
+  [(#3380)](https://github.com/PennyLaneAI/pennylane/pull/3380)
+
+  An example circuit that uses `FermionicSWAP` operation is:
+
+  ```python
+  dev = qml.device('default.qubit', wires=2)
+
+  @qml.qnode(dev)
+  def circuit(phi):
+      qml.BasisState(np.array([0, 1]), wires=[0, 1])
+      qml.OrbitalRotation(phi, wires=[0, 1])
```

Comment:
Should this be `qml.FermionicSWAP`? Or why are we doing `OrbitalRotation` here?

Reply:
Good catch! Fixing the typo.